content/setting-up-services/how-to-set-up-services/leverage-service-commands.md

```diff
@@ -1,13 +1,21 @@
 +++
 title = "Leverage Service Commands"
 date = 2019-09-19T13:03:56-04:00
+lastmod = 2020-07-20T15:00:00-04:00
 weight = 3
 +++
 
 While technically optional, Service Commands are arguably the most powerful part of the
 [configuration file](/setting-up-tugboat/create-a-tugboat-config-file/). This is the set of commands that Tugboat runs
 in a Service container while creating a Preview.
 
+Tugboat's service commands fit into two categories:
```

Comment:
I feel like `ready` and `clone` don't really fit well into these two categories. They are kind of a 3rd "other" category of some type.

`ready` runs both during and after preview builds

`clone` runs on clone, which is kind of like a build, but not really.